integration_tests/sdk/test-config-example.yml

```diff
@@ -0,0 +1,29 @@
+apikey: <APIKEY>
```

Comment:
Is there a place in documentation where are the required fields for each integration are laid out?

Reply:
For our internal dev, we should just be referencing the pydantic structs in `connect_config.py`. For users, I think adding some documentation would be good, although I don't know how much they would use something like that: https://linear.app/aqueducthq/issue/ENG-2238/add-documentation-for-connecting-integrations-from-the-sdk